docs/src/@primer/gatsby-theme-doctocat/components/live-preview-wrapper.js

```diff
@@ -1,4 +1,5 @@
-import {BaseStyles, Box, ThemeProvider, useTheme, DropdownMenu, DropdownButton} from '@primer/react'
+import {BaseStyles, Box, ThemeProvider, useTheme} from '@primer/react'
+import {DropdownMenu, DropdownButton} from '@primer/react/deprecated'
```

Comment:
👍. Would be nice to mop this up next week and refactor to ActionMenu+ActionList.

Reply:
Would there be anything stopping us from doing this?

Reply:
Nope, wanted to leave this here for next week :)

Reply:
👍 , let's add this to the v35 checklist then